index.css

```diff
@@ -0,0 +1,20 @@
+.forecast-header {
+    display: flex;
+    flex-direction: row;
+}
+
+.forecast-header .day {
+    flex-grow: 1;
+    text-align: left;
+}
+
+.forecast-header .date {
+    margin-right: 8px;
+}
+
+.forecast-header .refresh-button {
+    background-color: #009ad8;
+    color: white;
```

Comment:
Tu będzie problem z trybem dziennym i nocnym: zróbmy go w kolorze przycisku "find" na aktualnym mocku, bo jest pasujący dla dzień i noc (tj. #009ad8)

Nawet lepiej możnaby go zdefiniować jako button, żeby z css był zciągnięty wzór, ale to raczej wszystko jedno